src/containers/SearchPage/SearchPage.js

```diff
@@ -222,10 +228,11 @@ export class SearchPageComponent extends Component {
             searchParamsForPagination={parse(location.search)}
             showAsModalMaxWidth={MODAL_BREAKPOINT}
             primaryFilters={{
-              yogaStylesFilter: filters.yogaStylesFilter,
-              certificateFilter: filters.certificateFilter,
               priceFilter: filters.priceFilter,
+              dateRangeLengthFilter: filters.dateRangeLengthFilter,
               keywordFilter: filters.keywordFilter,
+              yogaStylesFilter: filters.yogaStylesFilter,
+              certificateFilter: filters.certificateFilter,
```

Comment:
The Yoga styles and Teacher certificate filters will be moved to secondary filters in another PR.

Reply:
@Gnito ^